packages/bar-loader/src/BarLoader.ts

```diff
@@ -12,19 +12,19 @@ governing permissions and limitations under the License.
 
 import {
     html,
-    LitElement,
+    SpectrumElement,
     CSSResultArray,
     TemplateResult,
     PropertyValues,
     property,
-} from 'lit-element';
+} from '@spectrum-web-components/base';
 
 import styles from './bar-loader.css.js';
 
 /**
  * @element sp-bar-loader
  */
-export class BarLoader extends LitElement {
+export class BarLoader extends SpectrumElement {
```

Comment:
Should the progress bar be mirrored for the `rtl` version of the determinate bar loader?

I checked out the `rtl` version of the bar loader in the [Spectrum CSS docs](https://opensource.adobe.com/spectrum-css/barloader.html) and noticed that progress bar runs right to left